core/src/test/scala/weaponregex/parser/ParserTest.scala

```diff
@@ -22,15 +22,17 @@ trait ParserTest {
 
   def treeBuildTest(tree: RegexTree, pattern: String): Unit = assertEquals(tree.build, pattern)
 
-  def parseErrorTest(pattern: String): Unit = {
-    val parsedTree = Parser(pattern, parserFlavor)
+  def parseErrorTest(pattern: String, flags: String): Unit = {
+    val parsedTree = Parser(pattern, flags, parserFlavor)
 
     assert(clue(parsedTree) match {
       case Failure(exception: RuntimeException) => exception.getMessage.startsWith("[Error] Parser:")
       case _                                    => false
     })
   }
 
+  def parseErrorTest(pattern: String): Unit = parseErrorTest(pattern, "")
```

Comment:
This can also be a default argument for flags in the other function

Reply:
Of course it should be 🤦‍♂️